Sources/Receipt/ListReceiptViewController.swift

```diff
@@ -68,7 +68,9 @@ public final class ListReceiptViewController: UITableViewController {
 
     // MARK: list receipt table view delegate
     override public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        // TODO will navigate to the recepit detail page
+        let hyperwalletReceipt = presenter.getHyperwalletReceiptAt(indexPath)
```

Comment:
you can directly use presenter.groupedSectionArray[indexPath.section].value[indexPath.row]. FYI groupedSectionArray has been renamed to sectionData. Pull the latest from epic branch to the lazy loading branch and from lazy loading to your branch and then update PR against epic branch instead of lazy loading

Reply:
done